README.md

```diff
@@ -704,6 +704,43 @@ If you are using a different terminal, proceed with manual font installation. 
      ```
      After changing the config run `xrdb ~/.Xresources` to reload it. The new config is applied to
      all new terminals.
+   - **[Crostini](https://chromeos.dev/en/linux)**: Open internal terminal preferences:
+    ```
+    chrome-untrusted://terminal/html/nassh_preferences_editor.html
+    ```
+    Set *Text font family* to:
+     ```text
+     'Source Code Pro', 'MesloLGS NF', 'Powerline For Source Code Pro', 'Noto Sans Mono'
```

Comment:
Just `'MesloLGS NF'` should be enough, right? Instructions for other terminals don't utilize fallback fonts (even though many terminals support them).

Reply:
My chromebox failed so I can't really test this again but from memory putting 'MesloLGS NF' first or alone didn't work :-(

I can try again when the chromebox fan is fixed.

Reply:
Let me know if you try again. Make sure to try with and without quotes.

Reply:
Works with quotes.

Reply:
> Works with quotes.

Thanks. I clarified this in https://github.com/romkatv/powerlevel10k/commit/0493886837595bdfb0d8ee36a8b25556ac0a64ea.